utils/src/main/kotlin/Extensions.kt

```diff
@@ -266,6 +261,11 @@ inline fun <K, V, W> Map<K, V>.zipWithDefault(other: Map<K, V>, default: V, oper
         operation(this[key] ?: default, other[key] ?: default)
     }
 
+/**
+ * Converts this [Number] from bytes to mebibytes (MiB).
```

Comment:
mebi -> mega

Reply:
No, that's actually correct, as this is about 1024, not 1000. Also see https://en.wikipedia.org/wiki/Mebibyte.

Reply:
Didn't know that term, interresting...